utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -0,0 +1,254 @@
+      associationsOverlay = new GraphicsOverlay();
+      associationsOverlay.setRenderer(uniqueValueRenderer);
+  
+      // create a map with the topographic vector basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createTopographicVector());
+      
+      // create a viewpoint to focus on the utility networks' extent
+      Viewpoint viewPoint = new Viewpoint(41.8057655, -88.1489692, 23);
+  
+      // set the initial viewpoint
+      map.setInitialViewpoint(viewPoint);
+      
+      // create a map view, set the map, and add the graphics overlay
+      mapView = new MapView();
+      mapView.setMap(map);
+      mapView.getGraphicsOverlays().add(associationsOverlay);
+      
+      // create the utility network
+      utilityNetwork = new UtilityNetwork("https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer");
+  
+      // load the utility network and get all of the edges and junctions in the network
+      utilityNetwork.loadAsync();
+      utilityNetwork.addDoneLoadingListener(() -> {
+        List<UtilityNetworkSource> networkSources = utilityNetwork.getDefinition().getNetworkSources();
+  
+        for (UtilityNetworkSource nSource : networkSources)
+          // add all edges that are not subnet lines to the map
+          if (nSource.getSourceType() == UtilityNetworkSource.Type.EDGE && nSource.getSourceUsageType() != UtilityNetworkSource.UsageType.SUBNET_LINE) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+          // add all junctions to the map
+          else if (nSource.getSourceType() == UtilityNetworkSource.Type.JUNCTION) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+  
+        // add association graphics at the initial view point
+        addAssociationsGraphics();
+        // listen for navigation changes
+        mapView.addNavigationChangedListener(event -> addAssociationsGraphics());
+      });
+  
+      // create a legend
+      GridPane gridPane = new GridPane();
+      gridPane.getStyleClass().add("grid-pane");
+      gridPane.getColumnConstraints().addAll(Arrays.asList(new ColumnConstraints(25), new ColumnConstraints(100)));
+      gridPane.add(attachmentImageView, 0, 0);
+      gridPane.add(attachmentLabel, 1, 0);
+      gridPane.add(connectivityImageView, 0, 1);
+      gridPane.add(connectivityLabel, 1, 1);
+      
+      // add the map view and legend to the stack pane
+      stackPane.getChildren().addAll(mapView, gridPane);
+      StackPane.setAlignment(gridPane, Pos.TOP_LEFT);
+      StackPane.setMargin(gridPane, new Insets(10, 0, 0, 10));
+      
+    } catch (Exception e) {
+      // on any error, display the stack trace
+      e.printStackTrace();
+    }
+    
+  }
+  
+  /**
+   * Get utility association type for each association within extent , create graphics and add it to the graphics overlay.
```

Comment:
just a minor adjustment to this comment here
```suggestion
   * Get the utility association type for each association within the extent, create graphics, and add it to the graphics overlay.
```